ci/scripts/check-bazel-tags.sh

```diff
@@ -6,14 +6,37 @@
 
 set -e
 
-untagged=$(./bazelisk.sh query "rdeps(//..., //hw:verilator) except attr(tags, '[\\[ ](verilator|manual)[,\\]]', //...)")
+untagged=$(./bazelisk.sh query "rdeps(//..., //hw:verilator) except attr(tags, '[\\[ ](verilator|manual)[,\\]]', //...)" --output=label_kind)
 if [[ ${untagged} ]]; then # Check that all targets that depend on verilator are tagged
-  echo "Target(s): ${untagged} depend(s) on //hw:verilator, please tag it with verilator or manual";
+  echo "Error:";
+  echo "${untagged}"|sed 's/^/    /';
+  echo "Target(s) above depend(s) on //hw:verilator, please tag it with verilator or manual.";
   exit 1
 fi
 
-untagged=$(./bazelisk.sh query "rdeps(//..., kind('bitstream_splice', //...)) except attr(tags, '[\\[ ](cw310_rom|cw310_test_rom|vivado|manual)[,\\]]', //...)")
+untagged=$(./bazelisk.sh query "rdeps(//..., kind('bitstream_splice', //...)) except attr(tags, '[\\[ ](cw310_rom|cw310_test_rom|vivado|manual)[,\\]]', //...)" --output=label_kind)
 if [[ ${untagged} ]]; then # Check that all targets that depend on vivado are tagged
```

Comment:
This comment might be a little misleading, could you also change it to say "all targets that depend on updatemem"? Targets like //hw/bitstream/vivado:fpga_cw310 also depend on Vivado.

Reply:
The rom depends on updatemem but probably shouldn't have the tag... We need to discuss the vivado tag further.